lib/socrates/core/dispatcher.rb

```diff
@@ -94,6 +97,8 @@ def do_dispatch(session, message)
             return
           end
 
+          @event_handler.call(session, :transition, msg: message, from: state.data.state_id, to: state.next_state_id)
```

Comment:
it's been a little while since I did a lot of ruby, ruby lets you drop the `{}` around objects right? So even though this looks like a long list of arguments it's really just three right? It's a session, an enum, and then an object with `msg` `from` and `to` keys right?